src/checkbox/Checkbox.tsx

```diff
@@ -27,12 +27,7 @@ const [CheckboxProvider, useCheckboxContext] = createContext<CheckboxContext>({
 
 export { useCheckboxContext };
 
-type CheckboxRenderProps = {
-  children?:
-    | ((state: ReakitCheckboxOptions) => JSX.Element)
-    | React.ReactNode
-    | string;
```

Comment:
Don't need `string` here i guess because React.ReactNode already contains `ReactText` which is `string | number`